.github/workflows/release.yml

```diff
@@ -446,9 +446,52 @@ jobs:
             process.exit(1);
           });
 
+  get-extra-tags:
+    needs: [ check-version ]
+    runs-on: ubuntu-22.04
+    outputs:
+      extra_tags: ${{ steps.taglist.outputs.extra_tags }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: release
+      - name: prepare release scripts
+        run: cd release && yarn && yarn build
+      - name: Get extra tags
+        id: taglist
+        uses: actions/github-script@v7
+        with:
+          result-encoding: string
+          script: | # js
+            const { getExtraTagsForVersion } = require('${{ github.workspace }}/release/dist/index.cjs');
+
+            const version = '${{ inputs.version }}';
+
+            const extra_tags = getExtraTagsForVersion({ version });
+
+            console.log({ extra_tags });
+            core.setOutput('extra_tags', JSON.stringify(extra_tags));
+
+  add-extra-tags:
```

Comment:
this spawns jobs like this:

![Screen Shot 2024-12-17 at 12 41 47 PM](https://github.com/user-attachments/assets/4cf2a140-8e04-4c00-8846-f17c54338e1c)